src/server.js

```diff
@@ -121,7 +116,7 @@ function index(req, res, next) {
 			pkg,
 			Helper.config
 		);
-		data.gitCommit = gitCommit;
+		data.gitCommit = Helper.getGitCommit();
```

Comment:
Can the frontend just use `Helper.getVersion()` too? Then the `getGitCommit` function is only internal and `getVersion` would be the single authority.